schemas/fragments/ReadStep.json

```diff
@@ -14,83 +14,116 @@
           "items": {
             "type": "string"
           },
-          "description": "A DDL-formatted schema. Schema can be used to coerce values into more appropriate data types."
+          "description": "A DDL-formatted schema. Schema can be used to coerce values into more appropriate data types.",
+          "examples": [
+            [
+              "date TIMESTAMP",
+              "city STRING",
+              "population INT"
+            ]
+          ]
         },
         "separator": {
           "type": "string",
-          "description": "Sets a single character as a separator for each field and value."
+          "description": "Sets a single character as a separator for each field and value.",
+          "default": ","
```

Comment:
Note @zaychenko-sergei that per our past discussion about representing these properties in UI I started populating default values. They are not currently used in codegen, but I ensured to synchronize them between Spark and DataFusion ingests.